packages/pwa-kit-mcp/src/utils/utils.js

```diff
@@ -106,3 +99,75 @@ export const getCreateAppCommand = () => {
           )
         : `@salesforce/pwa-kit-create-app@${CREATE_APP_VERSION}`
 }
+
+/**
+ * Runs an NPX command and captures its output.
+ *
+ * @returns {Promise<string>} - Resolves with the command output.
+ */
+export async function runNpxCommand(NPX_COMMAND, CREATE_APP_COMMAND, DISPLAY_PROGRAM_COMMAND) {
+    return new Promise((resolve, reject) => {
+        const tempDir = os.tmpdir()
+        const outputFilePath = path.join(tempDir, 'npx-output.json')
+        const errorFilePath = path.join(tempDir, 'npx-error.log')
+        const command = `${NPX_COMMAND} ${CREATE_APP_COMMAND} ${DISPLAY_PROGRAM_COMMAND} > ${outputFilePath} 2> ${errorFilePath}`
+
+        exec(command, (error) => {
+            if (error) {
+                reject(error)
+                return
+            }
+
+            fsPromises.promises
+                .readFile(outputFilePath, 'utf-8')
+                .then((data) => resolve(data))
+                .catch((err) => reject(err))
+        })
+    })
+}
+
+/**
+ * Returns the copyright header with the current year
+ * @returns {string} The copyright header text
+ */
+export const getCopyrightHeader = () => {
+    const year = new Date().getFullYear()
+    return `/*
+ * Copyright (c) ${year}, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */`
+}
```

Comment:
Will pages/component added by external developers have this header on their code?

This opened up the conversation about who is using this tool, if the tool is being used by salesforce internal developers it makes sense to add this header, but to my understanding the primary goal is for customer developers to use the mcp server, and they don't work for salesforce and won't want this header added to their code.